packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -87,9 +89,15 @@ function InteractionLayer ({ activeDrawingTask, activeTool, disabled, height, ma
         <DrawingToolMarks
           activeMarkId={activeMark && activeMark.id}
           marks={marks}
-          onDelete={() => setActiveMark(null)}
+          onDelete={() => {
+            setActiveMark_state(null)
+            setActiveMark(null)
+          }}
           onFinish={onFinish}
-          onSelectMark={mark => setActiveMark(mark)}
+          onSelectMark={mark => {
+            setActiveMark_state(mark)
+            setActiveMark(mark)
```

Comment:
Get rid of local state here too and set the active mark in the store, then reference it from the active drawing task.